src/Quests/CustomQuest.cs

```diff
@@ -33,7 +36,8 @@ public class CustomQuest
         public bool Cancelable { get; set; }
         public string ReactionText { get; set; }
         public int DaysLeft { get; set; } = 0;
-        public List<Hook> Hooks { get; set; }
+        public List<Hook> Hooks { get; set; }
+        public ConversationTopicOptions ConversationTopic { get; set; }

```

Comment:
Remove it, use events on `CustomQuest` instead. Keep it only in ContentPack model.